Lara-JS/scripts/build-interfaces.js

```diff
@@ -61,7 +61,9 @@ import {
   registerJoinpointMapper,
   wrapJoinPoint,
   unwrapJoinPoint,
-} from "lara-js/api/LaraJoinPoint.js";\n\n`
+} from "lara-js/api/LaraJoinPoint.js";
+import eventListener from "./clava/events/EventListener.js";
+import { Event, EventTime } from "./clava/events/Events.js";\n\n`
```

Comment:
The LARA framework should not know about the existence of Clava, which uses it. This is a sign that the events code currently in the Clava repository should be moved to the LARA framework repository, into the adequate folder.

I suggest moving moving the files to `lara-framework/Lara-JS/src-api/lara/events` and update imports accordingly.

Reply:
After looking at the code in [PR#157](https://github.com/specs-feup/clava/pull/157), it is not immediately possible to move the code the the lara-framework, it would need further refactoring in order to separate the Events interface from the Clava-specific implementation.